src/pages/developers/tutorials.tsx

```diff
@@ -0,0 +1,474 @@
+            )}
+          </Flex>
+        </Flex>
+        {filteredTutorials.length === 0 && (
+          <Box mt={0} textAlign="center" padding={12}>
+            <Emoji text=":crying_face:" fontSize="5xl" mb={8} mt={8} />
+            <OldHeading>
+              <Translation id="page-developers-tutorials:page-tutorial-tags-error" />
+            </OldHeading>
+            <Text>
+              <Translation id="page-developers-tutorials:page-find-wallet-try-removing" />
+            </Text>
+          </Box>
+        )}
+        {filteredTutorials.map((tutorial) => {
+          return (
+            <Flex
+              as={BaseLink}
+              textDecoration="none"
+              flexDirection="column"
+              justifyContent="space-between"
+              fontWeight="normal"
+              color="text"
+              boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
+              mb="px"
+              padding={8}
+              w="full"
+              _hover={{
+                textDecoration: "none",
+                borderRadius: "base",
+                boxShadow: "0 0 1px var(--eth-colors-primary-base)",
+                bg: "tableBackgroundHover",
+              }}
+              key={tutorial.to}
+              to={tutorial.to ?? undefined}
+              hideArrow
+            >
+              <Flex
+                justifyContent="space-between"
+                mb={{ base: 8, md: -4 }}
+                alignItems="flex-start"
+                flexDirection={{ base: "column", md: "initial" }}
+              >
+                <Text
+                  color="text"
+                  fontWeight="semibold"
+                  fontSize="2xl"
+                  mr={{ base: 0, md: 24 }}
+                  _after={{
+                    ml: 0.5,
+                    mr: "0.3rem",
+                    display: tutorial.isExternal ? "inline" : "none",
+                    content: `"↗"`,
+                    transitionProperty: "all",
+                    transitionDuration: "0.1s",
+                    transitionTimingFunction: "ease-in-out",
+                    fontStyle: "normal",
+                  }}
+                >
+                  {tutorial.title}
+                </Text>
+                <Badge variant="secondary">
+                  <Translation id={getSkillTranslationId(tutorial.skill!)} />
+                </Badge>
+              </Flex>
+              <Text color="text200" fontSize="sm" textTransform="uppercase">
+                <Emoji text=":writing_hand:" fontSize="sm" mr={2} />
+                {tutorial.author} •
+                {published(locale, tutorial.published ?? "")}
+                {tutorial.timeToRead && (
+                  <>
+                    {" "}
+                    •
+                    <Emoji text=":stopwatch:" fontSize="sm" ml={2} mr={2} />
+                    {tutorial.timeToRead}{" "}
+                    <Translation id="page-tutorial-read-time" />
```

Comment:
We need to specify the namespace in this case 
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/aeeea0cd-5726-4482-abca-9289cb4dd217)